### PR TITLE
klunker: basicrenewal of location klunker

### PR DESCRIPTION
**Pull request by Petrella**

### CODE REVIEW

File: group_vars/location_klunker/networks.yml [hunk 1]
```diff
@@ -1,34 +1,63 @@
...
+  - vid: 13
+    role: mesh_nnw
+    name: mesh_south
+    prefix: 10.31.71.155/32
+    ipv6_subprefix: -4
+    mesh_ap: klunker-ap2
+    mesh_radio: 11a_standard
+    mesh_iface: mesh
+    assignments:
+      klunker-core: 1
```
Comment (PolynomialDivision) [inline group_vars/location_klunker/networks.yml:51]: why do you add here an assignment?

File: group_vars/location_klunker/networks.yml [hunk 2]
```diff
@@ -1,34 +1,63 @@
...
     role: mesh
-    name: mesh_south
+    name: mesh_sse
     prefix: 10.31.71.154/32
     ipv6_subprefix: -3
     mesh_ap: klunker-ap1
     mesh_radio: 11a_standard
     mesh_iface: mesh
+    assignments:
+      klunker-core: 1
```
Comment (PolynomialDivision) [inline group_vars/location_klunker/networks.yml:38]: why do you add here an assignment?

File: group_vars/location_klunker/networks.yml [hunk 3]
```diff
@@ -38,10 +67,13 @@ networks:
     ipv6_subprefix: 0
     assignments:
       klunker-core: 1
+      klunker-ap1: 2
+      klunker-ap2: 3
+      klunker-ap3: 4
```
Comment (PolynomialDivision) [inline group_vars/location_klunker/networks.yml:72]: I would give them more reasoning names

Comment (PolynomialDivision) [reply group_vars/location_klunker/networks.yml:72]: Why do you have dhcp assignments?

Comment (PolynomialDivision) [reply group_vars/location_klunker/networks.yml:72]: Why do you add dhcp assignments?

File: group_vars/location_klunker/networks.yml [hunk 4]
```diff
@@ -50,5 +82,7 @@ networks:
       klunker-switch-1: 2
       klunker-rhnk: 3
       klunker-philmel: 4
-      klunker-ap1: 5
-      klunker-ap2: 6
+      klunker-rhnk-5ghz: 5
+      klunker-ap1: 6
+      klunker-ap2: 7
+      klunker-ap3: 8
```
Comment (PolynomialDivision) [inline group_vars/location_klunker/networks.yml:88]: More meaningful names would be good

File: host_vars/klunker-ap3/base.yml [hunk 5]
```diff
@@ -0,0 +1,8 @@
+---
+
+location: klunker
+role: ap
+model: "mikrotik_sxtsq-5-ac"
+mac_override:
+  eth0: 08:55:31:14:36:d7
+wireless_profile: freifunk_default
```
Comment (PolynomialDivision) [inline host_vars/klunker-ap3/base.yml:8]: this line is unncessary, but not harmful. freifunk_default is default for an ap (but not the core)

File: group_vars/location_klunker/networks.yml [hunk 6]
```diff
@@ -1,34 +1,63 @@
...
     mesh_ap: klunker-ap1
     mesh_radio: 11a_standard
     mesh_iface: mesh
+    assignments:
+      klunker-core: 1
+
+  # mesh down Erlanger Str.
+  # directing north-northwest
+  - vid: 13
+    role: mesh_nnw
```
Comment (PolynomialDivision) [inline group_vars/location_klunker/networks.yml:43]: that should be `mesh`. We don't have a rule called `mesh_nnw`.

**Comment by PolynomialDivision** [comment]
Please squash the two commits together.

**Comment by pktpls** [comment]
:smile_cat: 

- [ ] meaningful names such as `klunker-nf-s-5ghz`
- [ ] remove IP assignments on dhcp and mesh networks (except for klunker-core on dhcp)
- [ ] mesh role typo on vlan 13
- [ ] mesh_radio typo on vlan 14

File: group_vars/location_klunker/networks.yml [hunk 7]
```diff
@@ -1,17 +1,63 @@
...
+      klunker-core: 1
+
+  # 5GHz backup link to rhnk sector
+  - vid: 14
+    role: mesh
+    name: mesh_rhnk_5ghz
+    prefix: 10.31.71.156/32
+    ipv6_subprefix: -5
+    mesh_radio: 11a_standard
+    ptp: true
```
[the remark this answers is not in the document]
Comment (Petrella) [reply group_vars/location_klunker/networks.yml:56]: Can't be removed. It won't compile without and here it was not for the cube.

File: group_vars/location_klunker/networks.yml [hunk 8]
```diff
@@ -24,11 +66,16 @@ networks:
...
     role: mgmt
-    prefix: 10.31.71.144/29
+    prefix: 10.31.191.176/28
     gateway: 1
     dns: 1
     ipv6_subprefix: 1
     assignments:
       klunker-core: 1
-      klunker-poe: 2
+      klunker-switch-1: 2
```
Comment (PolynomialDivision) [inline group_vars/location_klunker/networks.yml:75]: Are there more than 1 switch?

Comment (Petrella) [reply group_vars/location_klunker/networks.yml:75]: not yet

**Comment by PolynomialDivision** [comment]
lgtm. lets wait for CI and than I can merge

**Pull request merged**